pkg/dao/repository_configs_test.go

```diff
@@ -1554,6 +1554,8 @@ func (suite *RepositoryConfigSuite) TestBulkDeleteRedhatRepository() {
 	orgID := config.RedHatOrg
 	repoConfigCount := 5
 
+	suite.tx.Exec("TRUNCATE repositories, snapshots, repositories_rpms, repositories_package_groups, repository_configurations")
```

Comment:
any idea why this is needed?

Reply:
i guess i should rephrase it, can we fix the test so this isn't needed?   I get its in a transaction and gets rolled back, so maybe its not a big deal, but it feels really bad

Reply:
that's fair. I can change that

Reply:
fixed